pkg/kubelet/kubelet.go

```diff
@@ -2892,6 +2892,58 @@ func (kl *Kubelet) canResizePod(pod *v1.Pod) (bool, v1.PodResizeStatus, string)
 	return true, v1.PodResizeStatusInProgress, ""
 }
 
+func IsMemoryResize(currentPod, allocatedPod *v1.Pod) bool {
```

Comment:
nit: doesn't need to be exported
```suggestion
func isMemoryResize(currentPod, allocatedPod *v1.Pod) bool {
```